README.md

```diff
@@ -66,6 +66,22 @@ uv run pytest <optional pytest args>
 Some tests require to compile solidity code, which requires `forge` to be
 installed, and `foundry` to be in the path, and to run `forge build`.
 
+#### Profiling tests
+
+To generate a profiling graph, you need to add `--profile-cairo` to your pytest
+command. For example:
+
+```bash
+$ uv run pytest -k get_u384_bits_little --profile-cairo
+...
+$ ls cairo/tests/ethereum/utils/test_numeric*.prof
```

Comment:
can you separate bash blocks?

```suggestion
ls cairo/tests/ethereum/utils/test_numeric*.prof
```

Reply:
Yes I can, do you think it would be more readable?